### PR TITLE
Fixes #80 versions of PKSIm now major

### LINKED ISSUE #80: Ensure that the version is depending on Major of PKSim

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/QualificationRunner.Core/QualificationRunnerConfiguration.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ private string getRegistryValueForRegistryPathAndKey(string openSystemsPharmacol
       {
          try
          {
-            return (string) Registry.GetValue($@"HKEY_LOCAL_MACHINE\SOFTWARE\{openSystemsPharmacology}{Version}", installDir, null);
+            return (string) Registry.GetValue($@"HKEY_LOCAL_MACHINE\SOFTWARE\{openSystemsPharmacology}{Major}", installDir, null);
```
Comment (msevestre) [inline src/QualificationRunner.Core/QualificationRunnerConfiguration.cs:52]: Now trying to find the version at the right place. 
I also updated the build to use the same structure as for v10

File: appveyor-nightly.yml [hunk 2]
```diff
@@ -1,18 +1,18 @@
 configuration: Debug
 image: Visual Studio 2019
 
-environment:
-  app_version: "9.1"
-  APPVEYOR_BLOCK_DOTNETCORE_TESTS_AUTORUN: true
+version: '{build}'
```
Comment (Yuri05) [inline appveyor-nightly.yml:4]: where is `version `used?

Comment (msevestre) [reply appveyor-nightly.yml:4]: This is a parameter that is used by appveyor

**Comment by msevestre** [comment]
@Yuri05 Merging so that I can run the build

**Pull request merged**